dtfs-central-api/src/v1/controllers/utilisation-report-service/helpers/get-record-correction-fields.ts

```diff
@@ -0,0 +1,59 @@
+import { format } from 'date-fns';
+import { FeeRecordEntity, mapReasonsToDisplayValues, FeeRecordCorrectionEntity, DATE_FORMATS } from '@ukef/dtfs2-common';
+import { getFormattedOldAndCorrectValues } from './get-formatted-old-and-correct-values';
+
+/**
+ * gets all the fields from a record correction or fee record in the correct format for a number of pages
+ * returns certain values as '-' if the correction is not completed
+ * @param feeRecord - the fee record
+ * @param correction - the fee record correction
+ * @returns formatted fields for the record correction and fee record
+ */
+export const getRecordCorrectionFields = (feeRecord: FeeRecordEntity, correction: FeeRecordCorrectionEntity) => {
```

Comment:
Please can we add a return type to this function to make it clearer what is/should be being returned? Should make it easier to maintain down the line.

Reply:
done